lib/client-api/src/story_store.js

```diff
@@ -261,13 +281,8 @@ export default class StoryStore extends EventEmitter {
       return null;
     }
 
-    const { story, parameters } = data;
-    return () =>
-      story({
-        kind,
-        story: name,
-        parameters,
-      });
+    const { story } = data;
+    return () => story();
```

Comment:
@ndelangen 

So now `getStory` and `getStoryWithContext` do actually the same thing. Should we deprecate one of those two?